sdk/sqlmanagement/Microsoft.Azure.Management.Sql/src/Generated/Models/Database.cs

```diff
@@ -225,7 +228,8 @@ public Database(string location, string id = default(string), string name = defa
             SecondaryType = secondaryType;
             CurrentSku = currentSku;
             AutoPauseDelay = autoPauseDelay;
-            StorageAccountType = storageAccountType;
```

Comment:
This a breaking change. Param order has change and StorageAccountType has been removed

Reply:
Discussed offline; this is something expected by customers currently using this feature